source/Locales/ScottLocale/ScottLocalesOfAlgebraicDcpos.lagda

```diff
@@ -0,0 +1,361 @@
+Ayberk Tosun.
+
+Started on 29 September 2023
+Finished on 2 October 2023.
+
+This module contains the definition of the Scott locale of a large and locally
+small _algebraic_ dcpo, using the definition of algebraic dcpo from the
+`DomainTheory` development due to Tom de Jong.
+
+If one starts with an algebraic dcpo, one can ensure that the resulting Scott
+locale is small by quantifying over only the basic/compact opens. This is the
+difference between the construction in this module and the one in
+`ScottLocale.Definition`
+
+TODO: in the future, it would be good to call the other module something
```

Comment:
Is this TODO done?

Reply:
No, see the discussion in [my response](https://github.com/martinescardo/TypeTopology/pull/201#issuecomment-1746602574) to @tomdjong's comments. I will do this in a future PR.

I have updated the comments to explain this in more detail.